src/BodyFunctionality/Body.java

```diff
@@ -167,6 +175,117 @@ public String getPlayer()
         return limbType;
     }
 
+    /**
+     * Method to determine if hands and toes have opposable digits
+     * @return  OpposableType: One of the enum OpposableTypes based on if hands or toes are opposable
+     */
+    public BodyPartGenerator.OpposableType getOpposableType()
+    {
+        boolean handsOpposable = false;
```

Comment:
This should go on the hand class, because with mishmashes you have no idea whatll happen